tests/test_ztf.rs

```diff
@@ -255,47 +265,112 @@ async fn test_process_ztf_lsst_xmatch() {
         .get_array("LSST")
         .unwrap();
     assert_eq!(matches.len(), 0);
-    drop_alert_from_collections(candid, "ZTF").await.unwrap();
```

Comment:
I'm curious why we don't want to drop.

Reply:
I mean we can, it's just not necessary. The point of the alert randomizers is to keep things unique so there is no conflict so dropping isn't a requirement. We can still drop stuff though, but now that we generate a bunch of entries with multiple candids, maybe the drop should rather take an objectId as parameter to drop everything related to the same object, that would be more practical.